index.html

```diff
@@ -301,6 +301,72 @@ <h3>
           </table>
         </section>
 
+        <section id="normative">
+          <h3>
+            Registries
+          </h3>
+          <p>
+The Working Group will deliver a set of registries on the [W3C Registry
+Track](https://www.w3.org/2021/Process-20211102/#registries) to
+support extension points in the above normative deliverables. The
+Working Group expects to produce the following registries, but it will
+adjust this list as needed to support the normative deliverables:
+          </p>
+          <table>
+            <tr>
+              <th>Registry</th>
+              <th>Description</th>
+              <th>Based on</th>
+            </tr>
+            <tr>
+              <td>
+Verifiable Credential Properties Registry
+              </td>
+              <td>
+A registry of extensions to properties used in the Verifiable Credentials Data
+Model specification.
+              </td>
+              <td>
+<a href="https://w3c-ccg.github.io/vc-extension-registry/">Verifiable Credential Extensions Registry</a>
```

Comment:
I shouldn't have to keep repeating myself about this.  "Verifiable Credential Extensions Registry" is both overly specific and almost certainly wrong, as there will be many kinds of extensions, each with its own registry.

It's probably best to stop trying to enumerate the registries in the charter.  Just say that creation of registries for WG-defined extension points is in scope and call it good.

Reply:
> I shouldn't have to keep repeating myself about this.

That document is the official registry for VC Extensions and the management of it was handed over to the W3C CCG [per official resolution](https://www.w3.org/2019/04/09-vcwg-minutes.html#resolution09) by the VC1WG. It has, for the last several years, with the involvement of the VCWG been called the "Verifiable Credential Extensions Registry". 

If the VCWG would like to change the name, it should resolve to do so. Given that it's an input document, and given that it's name is currently the "Verifiable Credential Extensions Registry", we should probably keep calling it that until the name of the document is changed.

As I noted previously, I have renamed the potential registry to "Verifiable Credential Properties Registry" per your request to change the target name. If you would like it to be changed to a different name, please suggest something concrete that will achieve consensus among the group.